INSTALLATION-es.md

```diff
@@ -2,8 +2,15 @@
 
 *[Read this in English](INSTALLATION.md)*
 
-## Software requerido
-- git, node, and an IDE
+Esta página documenta los pasos de instalación de algunos de los programas necesarios para trabajar con este proyecto.
+
+> **NOTA: Si todo lo que quiere hacer es ejecutar rápidamente todo localmente para probar la aplicación, vaya directamente a [`QUICKSTART.md`](QUICKSTART.md).**
+
+Después de los pasos de instalación genéricos de esta página,continúe con uno de los siguientes, según lo que intente hacer:
+
+- Si está trabajando con la interfaz, consulte [`client/README.md`](client/README.md).
```

Comment:
Are you available to, by chance, create a `client/README-es.md` and link to that file here?

Reply:
I am little bit confused,  can you please explain me , what I have to do ??

Reply:
Yes, thank you asking for clarification. 

Currently this `INSTALLATION-es.md` file has a link to the `client/README.md` file. This link is linking to this an English version of the README file. For the Spanish speaker, it would be better if the link led them to a Spanish version of the file, i.e., a README-es.md. 

The request is, for all the files that the INSTALLATION-es.md file links to, can we create a Spanish version of those files?

Tasks:
1. create a `client/README-es.md` that translates `client/README.md`
2. create a `data/data-pipeline/README-es.md` that translates `data/data-pipeline/README.md`
3. create a `.github/workflows/README-es.md` that translates `.github/workflows/README.md`
4. replace links in the INSTALLTION-es.md file with their Spanish equivalents (files from steps 1 -3)

Does that help shed light on the request?